LanguageFeatures/Extension-types/static_analysis_extension_types_A08_t07.dart

```diff
@@ -19,17 +19,18 @@
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=inline-class
-
 import 'dart:async';
 
-extension type NumET(num _) {}
-extension type IntET(int _) {}
-extension type FutureET<T>(Future<T> _) {}
+extension type NumET(num _) implements num {}
+extension type IntET(int _) implements NumET {}
 
-test1<X extends Future<NumET>>(X x) async {
+test1<X extends NumET>(X x) async {
   if (x is int) {
```

Comment:
We still can't promote to `int`, because that isn't a subtype of `NumET`.

Reply:
Indeed... Removed this